packages/pr-title-conventional-commits/action.yml

```diff
@@ -0,0 +1,71 @@
+name: Check the Pull Request title
+description: Check that the Pull Request title follow the 'Conventional Commits' requirements
+
+inputs:
+  github-token:
+    description: 'The token used to create Pull Request comment when the title does not follow requirements'
+    required: true
+    default: ${{ github.token }}
+  comment:
+    description: >
+      This input controls where the error messages and the fix suggestions are produced.
+      If 'true', create a Pull Request comment.
+      If 'false', add the messages in the GitHub Actions logs.
+      If 'auto', acts like 'false' for Pull Request created from forked repositories. Otherwise, acts like 'true'
+    required: true
+    default: 'auto'
```

Comment:
```suggestion    
    type: choice
    options:
      - default
      - true
      - auto
```

Reply:
This is only possible for workflows: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs